op-deployer/pkg/deployer/pipeline/opchain.go

```diff
@@ -137,7 +144,7 @@ func DeployOPChain(ctx context.Context, env *Env, bundle ArtifactsBundle, intent
 	var lastTaskErr error
 	for i := 0; i < len(setImplementationAddressTasks); i++ {
 		taskErr := <-errCh
-		if lastTaskErr != nil {
+		if taskErr != nil {
```

Comment:
@mslipper left a (now deleted?) comment that said:
> It's meant to be `lastTaskErr == nil` to capture the first error.

whereas this would capture the last error. Both seem ok to me, unsure if the difference matters, so will defer to slipper here

Reply:
this looks fine - i deleted it because the concept of "last error" was confuring.

Reply:
Will resolve this comment and keep current approach.